Headers/Public/VLCMediaPlayer.h

```diff
@@ -439,8 +439,8 @@ typedef NS_ENUM(unsigned, VLCMediaPlaybackSlaveType)
  */
 
 /**
- * Return the current chapter index, or
- * \return NSNotFound if none is set.
+ * Return the current chapter index
+ * \return current chapter index or -1 if there is no media
```

Comment:
not "no media" but "no chapter"

Reply:
I wanted it to be in sync with [libvlc](https://gyazo.com/06ed9c2c7b8f0e02a554f9a5600bfb3f).
 I do not mind changing it but would that mean a modification on libvlc-side as well?